nanogpt/download_json.py

```diff
@@ -0,0 +1,13 @@
+import requests
```

Comment:
This script doesn't seem necessary; it also doesn't need to parse and re-serialize the json data. It could be a single wget or curl command.